sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -80,6 +82,37 @@ export class EducationProgramOfferingService extends RecordDataModelService<Educ
     private readonly cancelAssessmentQueue: Queue<CancelAssessmentQueueInDTO>,
   ) {
     super(dataSource.getRepository(EducationProgramOffering));
+    this.applicationOfferingChangeRepo = dataSource.getRepository(
+      ApplicationOfferingChangeRequest,
+    );
+  }
+
+  /** Validates student authorization for the given education program offering.
+   * @param offeringId offering id.
+   * @param studentId student id.
+   * @returns true if the student is authorized for the given offering, otherwise false.
+   */
+  async validateApplicationOfferingForStudent(
```

Comment:
I believe this service should be inside `sources/packages/backend/apps/api/src/services/application-offering-change-request/application-offering-change-request.service.ts`

Reply:
https://github.com/bcgov/SIMS/pull/2196#discussion_r1297678987

Reply:
@sh16011993 The comment you tagged and this are about entirely different methods in 2 different files with the same name

Reply:
I see you did the change, so this comment is resolved. Thanks
![image](https://github.com/bcgov/SIMS/assets/77353155/8442f2d1-bbfb-4ed3-94c8-b6ceef8551ba)